content/_es/policy/rules-of-use.md

```diff
@@ -0,0 +1,164 @@
+
+Nunca compartiremos información con un socio sin su consentimiento explícito. Puede revocar este consentimiento en cualquier momento, y nosotros revocaremos el acceso del socio a la información. El socio podrá retenerla de acuerdo con sus políticas de retención de datos. El servicio de login.gov no recopila ninguna información que usted proporcione directamente al socio.
+
+Los posibles socios y otras partes interesadas pueden encontrar más información sobre el proceso de incorporación de socios [aquí](https://developers.login.gov/).
+
+# 9. Tarifas
+
+No se cobran tarifas al usuario por registrarse o utilizar el servicio de login.gov. No existe ninguna tarifa relacionada con la conexión con nuestros socios, aunque los propios socios pueden cobrar tarifas por algunos de sus servicios, como hacer una reserva, comprar un artículo o solicitar un servicio.
+
+# 10. Declaraciones, garantías y responsabilidades
+
+Renuncia a las garantías
+
+Los servicios de Login.gov se proporcionan &quot;tal cual&quot; y en función de su disponibilidad. Por la presente, GSA renuncia a todas las garantías de cualquier tipo, expresas o implícitas, incluidas, sin limitación, las garantías de comerciabilidad, idoneidad para un fin determinado y no infracción.
+
+GSA no garantiza que los servicios estén libres de errores o que el acceso a los mismos sea continuo o ininterrumpido.
+
+## 1. Declaraciones generales
+
+Por la presente, garantiza que (1) su uso del sitio web y de los servicios será estrictamente conforme al acuerdo y a todas las leyes y reglamentos aplicables, y (2) su uso del sitio web y de los servicios no infringirá ni se apropiará indebidamente de los derechos de propiedad intelectual de ningún tercero.
+
+## 2. Limitaciones de responsabilidad
+
+En ningún caso GSA será responsable con respecto a cualquier asunto de este acuerdo en virtud de cualquier contrato, negligencia, responsabilidad estricta u otra teoría legal o equitativa por (1) cualquier daño especial, incidental o consecuente; (2) el costo de adquisición de productos o servicios sustitutos; o (3) por la interrupción del uso o la pérdida o deterioro de datos.
+
+# 11. Disposiciones generales
+
+## 1. Acuerdo completo
+
+Este acuerdo constituye el acuerdo completo entre GSA y usted en relación con el uso del sitio web y los servicios, y solo se puede modificar mediante la publicación de una versión revisada en esta página por parte de GSA.
+
+## 2. Disputas
+
+Cualquier disputa que surja de este acuerdo y del acceso o uso de los servicios se regirá por la ley federal.
+
+## 3. Ausencia de renuncia a los derechos
+
+El hecho de que GSA no ejerza o haga cumplir cualquier derecho o disposición de este acuerdo no constituirá una renuncia a dicho derecho o disposición.
+
+## 4. Contacto
+
+Si tiene preguntas sobre estos términos o cualquier otro aspecto del servicio de login.gov, puede ponerse en contacto con nosotros en https://login.gov/contact/.
```

Comment:
linkify this?

```suggestion
Si tiene preguntas sobre estos términos o cualquier otro aspecto del servicio de login.gov, puede ponerse en contacto con nosotros en <https://login.gov/contact/>.
```

or maybe add in `{{site.baseurl}}/contact` or something with the right locale selector?